src/mlops_codex/training/validations.py

```diff
@@ -1 +1,14 @@
-# Specific validators for training
\ No newline at end of file
+# Specific validators for training
+# TODO: maybe it can be generic
+from mlops_codex.exceptions import InputError
+
+
+def validate_input(required, fields):
+    if (not all(k in fields for k in required)) or (
+        not all(fields[f] for f in required)
+    ):
+        raise InputError(
+            f"The parameters {required} are mandatory for custom trainings."
```

Comment:
Is this a generic function? If it is, this message is a bit especific.

Reply:
It is generic used by the classes `CustomTrainingExecution`, `AutoMLTrainingExecution` and `ExternalTrainingExecution`

https://github.com/datarisk-io/mlops_codex/pull/77/commits/e60a5d9b5c495d28f504ccd7fb2a04f6837e968f

Reply:
May be better no mention specific custom trainings then.

```suggestion
            f"The parameters {required} are mandatory for this training execution type."
```

Reply:
Thanks!